integration_tests/sdk/test-credentials-example.yml

```diff
@@ -3,10 +3,35 @@ apikey: <APIKEY>
 # Each integration entry starts with the name of the integration, followed
 # by the list of credentials needed to connect to it.
 data:
+  test_athena:
+    type: Athena
+    database: test
+    region: us-west-1
+    # We should all have access to this bucket, so no need to change the value here.
+    output_location: s3://aqueduct-athena-temp/tmp/
```

Comment:
Will this Athena thing go away with a rebase?

Reply:
This is what is currently on main: https://github.com/aqueducthq/aqueduct/blob/6b1319234fd4c3cfae427f28e4468207e54c8772/integration_tests/sdk/test-credentials-example.yml#L44
Should I remove it?